sycl/test-e2e/Config/allowlist.cpp

```diff
@@ -1,14 +1,16 @@
-// REQUIRES: cpu
+// REQUIRES: opencl && cpu
```

Comment:
The change in this file seems to be effectively NFC in comparison to what we've been testing before this PR.

Reply:
Why do we need this change?

Reply:
The test currently fails when running on a system with devices other than opencl. 

That is possibly an issue, but even if so, it has nothing to do with this change.

Reply:
Please, file a bug report. AFAIK, only OpenCL backend exposes CPU devices. I expect current version to work in our CI environment.

Reply:
https://github.com/intel/llvm/issues/9551